nle/env/base.py

```diff
@@ -521,6 +574,14 @@ def render(self, mode="human"):
         """
         chars_index = self._observation_keys.index("chars")
         if mode == "human":
+            obs = self.last_observation
+            tty_chars_index = self._observation_keys.index("tty_chars")
+            tty_colors_index = self._observation_keys.index("tty_colors")
+            tty_chars = obs[tty_chars_index]
+            tty_colors = obs[tty_colors_index]
+            self.tty_render(tty_chars, tty_colors)
```

Comment:
```suggestion
            rendering = self.get_tty_rendering(tty_chars, tty_colors)
            print(rendering)
```

Reply:
have moved this here because i think we wll want to reuse the conversion function in logging